tests/unit/Util/CSSTest.php

```diff
@@ -10,4 +16,20 @@ public function _before() {
   public function testItCanBeInstantiated() {
     expect_that($this->css instanceof \MailPoet\Util\CSS);
   }
+
+  public function testItParsesCssAndOrdersThemCorrectly() {
```

Comment:
Note: `div a` and `span a` have the same specificity, the resulting colors order depends on these rules order of appearance in the stylesheet.